releases/v1.1.1.json

```diff
@@ -0,0 +1,5 @@
+{
+    "milestone": "1.1.1",
+    "name": "BladeRunnerJS 1.1.1",
+    "prerelease": "false"
```

Comment:
@thecapdan This needs to be `false` rather than `"false"` - this is what was causing the error we saw last time we tried to release.